src/main/java/art/heredium/domain/membership/entity/CompanyMembershipRegistrationHistory.java

```diff
@@ -0,0 +1,70 @@
+package art.heredium.domain.membership.entity;
+
+import javax.persistence.*;
+
+import lombok.*;
+
+import com.vladmihalcea.hibernate.type.json.JsonStringType;
+import org.hibernate.annotations.DynamicInsert;
+import org.hibernate.annotations.TypeDef;
+
+import art.heredium.domain.account.entity.Account;
+
+@Entity
+@Getter
+@Table(name = "company_membership_registration_history")
+@DynamicInsert
+@TypeDef(name = "json", typeClass = JsonStringType.class)
+@NoArgsConstructor(access = AccessLevel.PROTECTED)
+@ToString
+public class CompanyMembershipRegistrationHistory {
+  @Id
+  @GeneratedValue(strategy = GenerationType.IDENTITY)
+  private Long id;
+
+  @Column(name = "title")
+  private String title;
+
+  @Column(name = "email_or_phone")
+  private String emailOrPhone;
+
+  @Column(name = "start_date")
+  private String startDate;
+
+  @Column(name = "price")
+  private Long price;
+
+  @Column(name = "payment_date")
+  private String paymentDate;
+
+  @Column(name = "status")
+  @Enumerated(EnumType.STRING)
+  private RegistrationStatus status;
+
+  @Column(name = "failed_reason")
+  private String failedReason;
+
+  @ManyToOne(fetch = FetchType.LAZY)
+  @JoinColumn(name = "account_id", nullable = false)
```

Comment:
```suggestion
  @JoinColumn(name = "account_id")
```
Sẽ có trường hợp ko match được với account_id (ví dụ emailOrPhone ko tìm thấy) nên trường này là nullable.

Reply:
Tôi đã fix: b45dbc1be37037c7c521a35ce6246929fa504153